modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -94,6 +103,59 @@ asmctl api create
 The API that was created is displayed.
 . Optionally, you can verify that an asset is created on Exchange and an API is created on API Manager.
 
+[[create-api-ASM-CLI-ConnAps]]
+=== Autocreate by Using Your Connected Apps Credentials
+
+If your users are managed by Anypoint Platform or an external IdP, use the Connected App credentials for authentication. Before you configure the autocreate feature, ensure that:
+
+* You xref:obtain-connected-apps-credentials.adoc[Obtain the client ID and client secret credentials] from Connected Apps
+* The ID of the group of assets on Exchange, for example, the organization ID
+* The client credentials has the:
+** Exchange Contributor permission for publishing the API to Exchange
+** Manage API Configuration permission for managing the API in API Manager
+
+To autocreate using Connected Apps, perform the following steps:
```

Comment:
parallel with previous:
```suggestion
To authenticate users to enable the autocreate feature, perform the following steps:
```

Reply:
To autocreate APIs using Connected Apps, perform the following steps:

Done! Thanks!